### ISSUE TITLE
Заменить непонятные API errors на соответствующие ошибки с нормальными названиями

### ISSUE THREAD

**Issue by albertos1607** [umerov1999/Fenrir-for-VK#9]
Например, если написать сообщение тому, кто меня заблокировал, появится сообщение об ошибке типа API.четотам.error, и то же если писать комментарии там, где они закрыты/доступны лишь по подписке. Думаю, стоит заменить эти названия на более понятные.

**Comment by albertos1607** [comment]
Поправочка: вылезает «dev.ragnarok.fenrir.api.ApiException».

**Comment by VladislavR34** [comment]
Согласен. Стоило бы это исправить. Иногда бывают вообще интересные ошибки, типа "!". А ты сиди и думай, что это
